patient-care/webapp/client/shared/editSampleGroup.js

```diff
@@ -335,8 +336,10 @@ Template.formValuesFilter.onCreated(function(){
   instance.editing = new ReactiveVar(false);
 
   let dataset_id = instance.data.data_set_id ;
-  instance.available_filter_forms = new ReactiveVar(); 
-  instance.available_filter_forms.set([{name: "Loading forms...", formId: "placeholder_loadingforms"}]);
+  instance.available_filter_forms = new ReactiveVar();
+  instance.available_filter_forms.set([{name: "Loading forms...", urlencodedId: "placeholder_loadingforms"}]);
```

Comment:
This file has a bad merge!

`urlencodedId` is an old version that was superseded by changes [here](https://github.com/UCSC-MedBook/MedBook_/pull/29/files/a099dfa926a60d728d31b77944fefe324f844ab6..16dcfe79807581c8657807b64cb00c4a87d4eb93).  

Looks like the change was reverted by [this](https://github.com/UCSC-MedBook/MedBook_/commit/f537fbaf59430a83674789d3ee408ae845d45284#diff-30b3fd25f7e664ae3f66c73044c0590c)  pull from the original patient-care.

We'll need to go through and make sure that we didn't lose any other older changes to master from that or other merges...